var/spack/repos/builtin/packages/hipblaslt/package.py

```diff
@@ -28,6 +28,7 @@ class Hipblaslt(CMakePackage):
 
     depends_on("c", type="build")
     depends_on("cxx", type="build")
+    depends_on("cmake@3.25.2:", type="build", when="@6.3.0:")
```

Comment:
```suggestion
    depends_on("cmake@3.25.2:", type="build", when="@6.2.0:")
```
The 6.2 versions also require this